<?xml version="1.0"?>
<launch>
  <arg name="target" default="localhost"/>
  <arg name="ns" default=""/>
  <arg name="unit" default="sensor0"/>

  <machine name="localhost"  address="localhost" />
  <machine name="remote2" address="remote2" env-loader="/home/pi/catkin_ws/devel/env.sh"  user="pi"  password="raspberry"/>
  <machine name="remote3" address="remote3" env-loader="/home/pi/catkin_ws/devel/env.sh"  user="pi"  password="raspberry"/>
  <machine name="OptiPlex-9010" address="OptiPlex-9010" env-loader="/home/erio/catkin_ws/devel/env.sh" user="erio" password="419000"/>

<!--
  <node machine="$(arg target)" name="uvc_camera_node" pkg="uvc_camera" type="uvc_camera_node">
    <param name="width"  value="800"/>
    <param name="height" value="600"/>
  </node>
-->
  <node name="image_rosbag" pkg="rosbag" type="play" args="$(find s4_sensor)/resources/single_coke.bag -s 20"/>

  <node name="image_republish" pkg="image_transport" type="republish" args="compressed raw">
    <remap from="in" to="/image_raw" />
    <remap from="out" to="/image_exp" />
  </node>
  
  <node name="image_view" pkg="image_view" type="image_view" >
    <remap from="image" to="/image_exp"/>
  </node>

  <node name="coke_detect" pkg="s4_sensor" type="coke_detect.py">
    <remap from="input_image" to="/image_exp" />
  </node>

  <node name="rects_tracker" pkg="s4_sensor" type="rects_tracker.py" output="screen">
    <remap from="input_image" to="/image_exp" />
  </node>

  <node name="image_view2" pkg="image_view" type="image_view" >
    <remap from="image" to="/output_image"/>
  </node>
</launch>

